SimMuon/RPCDigitizer/python/muonRPCDigis_cfi.py

```diff
@@ -22,6 +22,7 @@
         timeJitter = cms.double(1.0),
         IRPC_time_resolution = cms.double(0.1),
         IRPC_electronics_jitter = cms.double(0.025)
```

Comment:
I forgot a comma here at the end of the line. this creates the crash in the test.